### DISCUSSION TITLE
Thank you for this release!

### DISCUSSION THREAD

**Discussion by PoloDePoli** [maforget/ComicRackCE#9] [General]
Just wanted to thank you for this release. I look forward to trying it out and to what the community will achieve. 

Link to the Reddit thread: https://www.reddit.com/r/comicrackusers/s/WlpKQkOB3b

**Comment by ghost** [comment]
Thanks Maforget!  Looks like your really working hard on this.  We sooo appreciate it!!  People like me with large collections have been waiting years for this.

**Comment by tdisalvo** [comment]
Thank you so much for the work which is being done keeping this amazing product alive.
